ods_ci/tasks/Resources/Files/dsc_template.yml

```diff
@@ -41,6 +41,7 @@ spec:
     workbenches:
       devFlags: <workbenches_devflags>
       managementState: <workbenches_value>
+      workbenchNamespace: <workbenches_namespace>
```

Comment:
IUUC by doing this we no longer verify what happens when this is left unset. Are we ok with that?